### ISSUE TITLE
How long does populating step take?

### ISSUE THREAD

**Issue by jianshu93** [WrightonLabCSU/DRAM#345]
Hello team,

For many of the database preparation steps, things are not clear:

2024-05-03 05:18:56,565 - Populating the description db, this may take some time

What is it may take some time? A day is some time, a week is also some time. I would be more useful when using numbers. By the way, why do you use python to do the database population step, sqlite is in C++, which could be much faster than a python API, especially when we have such large database. 

Thanks,

Jianshu

**Comment by sahilrishav2** [comment]
I agree, i have the same issues but in my case it got killed automatically

**Comment by jianshu93** [comment]
It has been 30+ hours now, still no idea when it will finish. Another solution will be providing and pre-built database, which is much more convenient than building it from scratch. No idea why this option is not available.

Jianshu

**Comment by BioRRW** [comment]
Thank you for choosing to use DRAM. Unfortunately, at this time, we do not have the capacity to provide support for DRAM. We appreciate your understanding and encourage you to stay tuned for the release of DRAM2, expected in 2025.

Best regards, DRAM dev team

**Comment by sahilrishav2** [comment]
What is the memory size of your system, it should be more than 512GB.
